docs/conf.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python3
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath(".."))
+from bth5 import __version__
```

Comment:
It is probably better to set this as a string and then let rever do the replacement (ie add a like to `rever.xsh`)